### PR TITLE
add TF logo to Readme.md

### PR DESCRIPTION
**Pull request by iBrandyJackson**
### Description

adds TF logo to README in order to better match other HC official providers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<a href="https://terraform.io">
+    <img src=".github/tf.png" alt="Terraform logo" title="Terraform" align="left" height="50" />
```
Comment (tenthirtyam) [inline README.md:2]: I'm not sure that this path will work. May need to place in github.com/hashicorp/.github repo and then get the url.

Comment (tenthirtyam) [reply README.md:2]: image

Comment (tenthirtyam) [reply README.md:2]:
```html
<a href="https://terraform.io">
    <img src=".github/terraform_logo.svg" alt="Terraform logo" title="Terraform" align="right" height="50" />
</a>
```

Add `terraform_logo.svg` file from https://github.com/hashicorp/terraform-provider-aws/blob/main/.github/terraform_logo.svg

Comment (iBrandyJackson) [reply README.md:2]: Thanks for taking a look at this @tenthirtyam - appreciate it! I indeed failed to add in the .png file! I have fixed this - to note, we are using a different logo than the one you proposed

Comment (tenthirtyam) [reply README.md:2]: 🎉 nice addition!

**Review by appilon — APPROVED** [review]

**Pull request merged**